### ISSUE TITLE
[BUG] Markdown table overflow page's right border in DOCX

### ISSUE THREAD

**Issue by VincentTam** [jgm/pandoc#7407]
**Explain the problem.**

Steps used to reproduce the problem:

1. Create a sample Markdown file `test.md` with four lines.

    ```
    | col1 | col2 |
    | :--- | :--- |
    | abc | def |
    | abc | def |
    ```

1. Run this command in Vim to convert this Markdown file to DOCX.

       :exec '!pandoc % -o ' . expand('%:r') .'.docx'

    :information_source: In Vim,

    - `%` is expanded to the current file's path.
    - [`%:r`](https://vimhelp.org/cmdline.txt.html#filename-modifiers) takes away the extension name.
    - The operator `.` concatenates strings.
    - `!` runs external shell command.

1. Open LibreOffice to observe the rendered Markdown table.  The left column overflows the page's right border and the right column is hidden.

    ![screenshot of rendered Markdown table](https://user-images.githubusercontent.com/5748535/123237385-edf43e80-d4dd-11eb-8918-49d95781d847.png)

Expected behavior:

![two equal columns entirely contained inside the page](https://user-images.githubusercontent.com/5748535/123239491-b2f30a80-d4df-11eb-9273-b24a69b1651f.png)

The entire table with two columns with identical widths shall be contained inside the page.

**Pandoc version?**

| software/OS | version |
| --- | --- |
| Xubuntu | 20.04 |
| pandoc | 2.13 |
| Vim | 8.1 |
| LibreOffice | 6.4.7.2 |

**Comment by argent0** [comment]
I couldn't reproduce in 2.14.0.2

```
 $ pandoc -f markdown -o foo.docx
| col1 | col2 |
| :--- | :--- |
| abc | def |
| abc | def |
```

![table](https://user-images.githubusercontent.com/691306/123258049-4ff67900-d4c9-11eb-99d3-1eb24d31480f.png)

**Comment by tarleb** [comment]
Duplicate of #2576 and fixed via 0921b82d98b6ec7fa80ffd522c129b3828b9c00b, which was released with pandoc 2.14.

**Comment by VincentTam** [comment]
Thanks for response.  I'll update pandoc later.
